crates/frontend/src/pages/default_config/default_config.rs

```diff
@@ -0,0 +1,386 @@
+use crate::api::fetch_default_config;
+use crate::components::default_config_form::default_config_form::DefaultConfigForm;
+use crate::components::drawer::drawer::{close_drawer, open_drawer, Drawer, DrawerBtn};
+use crate::components::stat::stat::Stat;
+use crate::components::table::{table::Table, types::Column};
+use crate::types::BreadCrums;
+use leptos::*;
+use leptos_router::{use_navigate, use_query_map};
+use serde_json::{json, Map, Value};
+use std::collections::{HashMap, HashSet};
+
+#[derive(Clone, Debug, Default)]
+pub struct RowData {
+    pub key: String,
+    pub value: String,
+    pub pattern: String,
+    pub type_: String,
+    pub function_name: Option<Value>,
+}
+
+#[component]
+pub fn default_config() -> impl IntoView {
+    let tenant_rs = use_context::<ReadSignal<String>>().unwrap();
+    let default_config_resource = create_blocking_resource(
+        move || tenant_rs.get(),
+        |current_tenant| async move {
+            match fetch_default_config(current_tenant).await {
+                Ok(data) => data,
+                Err(_) => vec![],
+            }
+        },
+    );
+
+    let selected_config = create_rw_signal::<Option<RowData>>(None);
+    let key_prefix = create_rw_signal::<Option<String>>(None);
+    let enable_grouping = create_rw_signal(false);
+    let query_params = use_query_map();
+    let bread_crums = Signal::derive(move || {
+        let key_prefix = key_prefix.get();
+        let mut default_bread_crums = vec![BreadCrums {
+            key: "Default Config".to_string(),
+            value: None,
+            is_link: true,
+        }];
+
+        let mut bread_crums = match key_prefix {
```

Comment:
can we have different variable name `bread_crums` ?

Reply:
bread_crums name serves the purpose.